srcpkgs/containers.common/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'containers.common'
+pkgname=containers.common
+version=0.55.2
+revision=1
+hostmakedepends="go-md2man"
+short_desc="Docs and configs shared by podman, buildah, and skopeo"
+maintainer="Cameron Nemo <cam@nohom.org>"
+license="Apache-2.0"
+homepage="https://github.com/containers/common"
+distfiles="https://github.com/containers/common/archive/v${version}.tar.gz"
+checksum=997529c8aed1b6b71ff732d0cb75e67560222012402a3715ccab765b92ce0479
+
+do_build() {
+	DESTDIR=$DESTDIR make -C docs
```

Comment:
Is there a reason to not use `build_style=gnu-makefile` with `make_build_args="-C docs"`